documentation/man/features/report.rst

```diff
@@ -39,5 +39,59 @@ OPTIONS
 \--year [<year>]:
   Exhibits the current year summary if the user does not specify *<year>*.
 
+\--statistics:
```

Comment:
This commit should be squashed with the one that enables the `--statistic` option inside the report.